hendridg/metrics-webapp
# Testing

# :bookmark_tabs: Testing components :factory_worker: 

- Add complete testing to App.js, check to render components and testing reducers
- `test-utils.js` to config redux, and `App.test.js` to test app.
-  `details.test.js` and `home.test.js` to test reducers.
- Update title and description to `App`

<img src="https://media2.giphy.com/media/KbvZsN07K9Hy9ZoyR7/giphy.gif"/>
